internal/cluster/cluster_operations.go

```diff
@@ -390,9 +390,7 @@ func ImportCluster(ctx context.Context, data *schema.ResourceData, apiClient *cl
 		WithTimeout(constants.DefaultVcfApiCallTimeout)
 	getClusterParams.ID = clusterId
 	clusterResult, err := apiClient.Clusters.GetCluster(getClusterParams)
-	if err != nil {
-		return nil, err
-	}
+
```

Comment:
This should be added back. 

```suggestion
    if err != nil {
		return nil, err
	}

```